src/platforms/hosted/jlink.c

```diff
@@ -302,7 +304,7 @@ static bool jlink_get_version(void)
 
 		jlink.hw_version = read_le4(buffer, 0);
 
-		DEBUG_INFO("Hardware Version: %s V%u.%u.%u\n",
+		DEBUG_INFO("Hardware Version: %s v%u.%u.%u\n",
```

Comment:
"Hardware Version" => "Hardware version" for consistency with the other version print please.